packages/devtools_app/lib/src/screens/vm_developer/vm_class_display.dart

```diff
@@ -12,6 +12,9 @@ import '../../shared/theme.dart';
 import 'vm_developer_common_widgets.dart';
 import 'vm_object_model.dart';
 
+//TODO(mtaylee): Finish implementation of ClassInstancesWidget. When done,
```

Comment:
ubernit: comments should always be followed by a space before the actual comment message.